examples/src/main/java/com/hedera/hashgraph/sdk/examples/ContractHelper.java

```diff
@@ -59,7 +59,7 @@ public class ContractHelper {
     final Map<Integer, AccountId> stepFeePayers = new HashMap<>();
 
     public static String getBytecodeHex(String filename) throws IOException {
-        try (InputStream jsonStream = ContractHelper.class.getClassLoader().getResourceAsStream(filename)) {
+        try (InputStream jsonStream = ContractHelper.class.getResourceAsStream(filename)) {
```

Comment:
Production code change: resources need to be loaded "inside the module" when this runs on the module path.